Sources/TuistGraph/Models/Settings.swift

```diff
@@ -95,6 +95,8 @@ extension DefaultSettings {
     }
 }
 
+public typealias BuildConfigurationDictionary = [BuildConfiguration: TuistGraph.Configuration?]
```

Comment:
```suggestion
public typealias BuildConfigurationDictionary = [BuildConfiguration: Configuration?]
```

Reply:
The problem is Xcode opening the wrong Configuration on click for me, this is solving the problem and dies not hurt right?

Reply:
Makes sense, see https://github.com/tuist/tuist/pull/5992#discussion_r1502217993